src/styles/main.scss

```diff
@@ -1,3 +1,78 @@
+$days-amount: 31;
+$day-block-size: 100px;
+$calendar-gap: 1px;
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+@mixin calendar-styles {
+  display: flex;
+  flex-wrap: wrap;
+  width: 6 * $calendar-gap + 7 * $day-block-size;
+  gap: $calendar-gap;
+  padding: 10px;
+}
+
```

Comment:
All variables should be kept in a separate file for better readability and reuse.